Test/selenium_test.py

```diff
@@ -27,8 +27,8 @@ def driver():
 def test_app():
     # new board created by calling trello from init file under Test
     #board_id = Trello().create_test_board("Tasks_Selenium")
-    new_board = json.loads(board)
-    board_id = new_board["id"]
+    # new_board = json.loads(board) - Removed Feb 22 post Mongo
```

Comment:
Let's delete these lines rather than commenting them. Also, do we want to do something equivalent, e.g. if we're using the same Mongo cluster, we could use a separate database to avoid polluting our real data with test data (and vice versa)